config/config.go

```diff
@@ -97,6 +100,24 @@ func (c *Config) Save() {
 	c.RUnlock()
 }
 
+func (c *Config) IsUniqPeerAlias(alias string) bool {
+	c.Lock()
```

Comment:
Why not RLock?

Reply:
I forget to change it when moved on to no-cache solution. Thanks!